### PR TITLE
chore: old migration

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

File: core/migrations/1666463689317-Automigration.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Automigration1666463689317 implements MigrationInterface {
+    name = 'Automigration1666463689317'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "user" DROP COLUMN "type"`);
+        await queryRunner.query(`ALTER TABLE "history"."user_history" DROP COLUMN "type"`);
```
Comment (0c370t) [inline core/migrations/1666463689317-Automigration.ts:8]: Why are we touching history tables here?

Comment (bwpikaard) [reply core/migrations/1666463689317-Automigration.ts:8]: Type is irrelevant and was never used.

Comment (bwpikaard) [reply core/migrations/1666463689317-Automigration.ts:8]: It was something Nigel added in long ago but we ended up using MLE Org Teams.

Comment (0c370t) [reply core/migrations/1666463689317-Automigration.ts:8]: We still shouldn't drop columns from history tables regardless

**Pull request merged**